doc/attributes.md

```diff
@@ -8,35 +8,55 @@ The YaST UI for creating and editing a user offers the following attributes:
 
 | UI Field | Y2Users attr | Only when adding | Only when editing | Description |
 | :--- | :--- | :--- | :--- | :--- |
-| full name | gecos | | | |
-| name | name | | | |
-| password | password | | | |
-| system mail | | | | |
+| full name | `#gecos` | | | |
+| name | `#name` | | | |
+| password | `#password` | | | |
+| system mail | `#receive_system_mail` | | | See `MailAliases` module |
 | disable login | * password starting by `!` | | | |
-| uid | uid | | | |
-| home dir | home | | | | |
-| home dir permission | | | | |
-| empty home | | yes | | |
-| Move to new location | | | yes |  |
-| Btrfs subvolume | btrfs_subvolume_home | | | |
-| Additional info | gecos | | | |
-| login shell | shell | | | |
-| group | primary_group | | | |
-| additional groups | groups | | | |
-| ssh public keys | authorized_keys | | | |
-
-### Home management
-
-### Creating a user
+| uid | `#uid` | | | |
+| home dir | `Home#path` | | | | |
+| home dir permission | `Home#permissions` | yes | | `useradd -K HOME_MODE=0755` |
+| empty home | `CommitConfig#home_without_skel` | yes | | `rm -rf` after creating. No way to ignore `/usr/etc/skel` |
+| Move to new location | `CommitConfig#move_home` | | yes |  |
+| Btrfs subvolume | `Home#btrfs_subvol` | | | |
```

Comment:
this only when adding and only when edditing is kind of strange for me, as it should be exclusive. What having just one collumn having there Add/Edit/All ?